README.md

```diff
@@ -9,11 +9,29 @@ similar to scikit-learn. The library also makes it easy to backtest models,
 and combine the predictions of several models and external regressors.
 
 # Install
+## Preconditions
+Our direct dependencies include `fbprophet` and `torch` which have non-Python dependencies.
+A Conda environment is thus recommended because it will handle all of those in one go.
 
-From your favorite Python environment (3.6+), you can run:
-```
-pip install u8timeseries
-```
+The following steps assume running inside a conda environment. 
+If that's not possible, first follow the official instructions to install 
+[fbprophet](https://facebook.github.io/prophet/docs/installation.html#python)
+and [torch](https://pytorch.org/get-started/locally/), then skip to 
+[Install u8timeseries](#install-u8timeseries)
+
+To create a conda environment for Python 3.7
```

Comment:
I would put 3.6 as it is our min requirement.

Reply:
sure, though it's just an example for users that never used conda